### PR TITLE
Listing Page - reinstate 'month' in 'Joined Since...'

### PR DESCRIPTION
**Pull request by jerminatorhits**
## Description
Why did you write this code?
and styles
image
should look like 
image

## How to Test
- [ ] Visit listings/:id
- [ ] Click Y

Which devices did you test on?
- [x] Chrome on Mac
- [ ] Chrome on PC
- [ ] Firefox on Mac
- [ ] Firefox on PC
- [ ] Safari iPhone
- [ ] Chrome Android

## REVIEWERS:
Check against these principles:

### High level
Does this code need to be written?
What are the alternatives?
Will this implementation become a support issue?
How much error margin does this solution have?

### Code
* Does the code follow industry standards?
JS: https://github.com/airbnb/javascript
React: https://github.com/airbnb/javascript/tree/master/react
https://github.com/vasanthk/react-bits
Documentation headers: http://usejsdoc.org/index.html

* Is there duplicated code? Can it be refactored into a shared method?
* Is the code consistent with our project?
* Are there unit tests? Do they test the states?
* Is the person refactoring another developer's code? If possible, did the original developer approve?

### Variables/Naming:
* Would the variable type led to future edge cases?
* Are the variable naming clear? Would the value contain something other than what the name describes.

### Security
* Can this be hacked or abused by the user?

## Further Work
Will this have future work?

## Learnings
Did you learn anything here you want to share with the team?

### CODE REVIEW

File: src/components/routes/Listing/Listing/ListingInformation.tsx [hunk 1]
```diff
@@ -9,11 +9,11 @@ import GoogleMaps from 'shared/GoogleMaps';
 import LazyImage from 'shared/LazyImage';
 import ListItem from 'shared/ListItem';
 import Svg from 'shared/Svg';
-import { dateToYear } from 'utils/formatDate';
 import { formatAddress  } from 'utils/formatter'
 import Portal from 'components/shared/Portal';
 import ContactHostForm from 'components/shared/ContactHostForm';
 import { ToggleProviderRef, ToggleProvider } from 'components/shared/ToggleProvider';
+import { format } from 'date-fns';
```
Comment (kcvan) [inline src/components/routes/Listing/Listing/ListingInformation.tsx]: i think you can cherry pick by doing `import format from 'import format from `date-fns/format';`

Comment (jerminatorhits) [reply src/components/routes/Listing/Listing/ListingInformation.tsx]: 🙏
